Model/Api/Debug.php

```diff
@@ -178,16 +202,16 @@ public function debug()
 
     /**
      * This method will handle universal api debug requests based on the type of request it is.
-     * 
+     *
      * @param array $data
      * @return \Bolt\Boltpay\Api\Data\DebugInfo
      * @throws BoltException
      * **/
     public function universalDebug($data){
```

Comment:
Please add new data into this function as well. Actually we use only this one.

Reply:
updated